src/iden3comm/handlers/auth.ts

```diff
@@ -185,38 +221,17 @@ export class AuthHandler implements IAuthHandler {
     return authRequest;
   }
 
-  /**
-   * @inheritdoc IAuthHandler#handleAuthorizationRequest
-   */
-  async handleAuthorizationRequest(
-    did: DID,
-    request: Uint8Array,
-    opts?: AuthHandlerOptions
-  ): Promise<{
-    token: string;
-    authRequest: AuthorizationRequestMessage;
-    authResponse: AuthorizationResponseMessage;
-  }> {
-    const authRequest = await this.parseAuthorizationRequest(request);
-
+  private async handleAuthRequest(
+    authRequest: AuthorizationRequestMessage,
+    ctx: AuthReqOptions
+  ): Promise<AuthorizationResponseMessage> {
     if (authRequest.type !== PROTOCOL_MESSAGE_TYPE.AUTHORIZATION_REQUEST_MESSAGE_TYPE) {
       throw new Error('Invalid message type for authorization request');
     }
 
-    if (!opts) {
-      opts = {
-        mediaType: MediaType.ZKPMessage
-      };
-    }
-
-    if (opts.mediaType === MediaType.SignedMessage && !opts.packerOptions) {
-      throw new Error(`jws packer options are required for ${MediaType.SignedMessage}`);
-    }
-
-    if (authRequest.to) {
-      // override sender did if it's explicitly specified in the auth request
-      did = DID.parse(authRequest.to);
-    }
+    // override sender did if it's explicitly specified in the auth request
+    const did = authRequest.to ? DID.parse(authRequest.to) : ctx.did;
```

Comment:
In comment above: "override sender did ...", but here you are overriding target did.